tests/test_selected_schema.py

```diff
@@ -0,0 +1,43 @@
+"""Tests standard tap features using the built-in SDK tests library."""
+import json
+
+import pendulum
+from singer_sdk.testing.templates import TapTestTemplate
+
+from tap_postgres.tap import TapPostgres
+
+TABLE_NAME = "test_replication_key"
+SAMPLE_CONFIG = {
+    "sqlalchemy_url": "postgresql://postgres:postgres@localhost:5432/postgres",
+}
+
+
+def selected_schema_test(tap, table_name):
+    """excluding one column from stream and check if it is not present in query"""
+    column_to_exclude = "name"
+    tap.run_discovery()
+    tap_catalog = json.loads(tap.catalog_json_text)
```

Comment:
Seems like we're relying on another test to populate the table? We should just populate the DB with the table we care about and then just pull that table here so the test is isolated.